### ISSUE TITLE
Outbound Method override not working

### ISSUE THREAD

**Issue by cwiechmann** [Axway-API-Management-Plus/apim-cli#50]
**API-Mgr. CLI version**
1.2.0

**API-Manager and Service-Pack Version**
7.6.2 SP4

**Expected behavior**
When configuring a method override in the "Outbound" section like so:
```json
  "outboundProfiles" : {
    "_default" : {
      "routeType" : "proxy",
      "authenticationProfile" : "_default"
    },
    "deleteOrder" : {
      "routeType" : "proxy",
      "requestPolicy" : "Request policy 1"
    }
  },
```
The API is successfully replicated but when reviewing that API in the UI is shows the following error (in 7.6.2 SP4):
![image](https://user-images.githubusercontent.com/16292002/88031431-e04f6f80-cb3c-11ea-847a-0d7594facaa3.png)

See issue #39 for more details.

**Comment by cwiechmann** [comment]
SNAPSHOT version including this fix:
[axway-apimcli-1.3.0-SNAPSHOT.zip](https://github.com/Axway-API-Management-Plus/apim-cli/files/4952499/axway-apimcli-1.3.0-SNAPSHOT.zip)

**Comment by jschaefer77** [comment]
Hi @cwiechmann, I tested if with API Manager 7.6.2 SP and V7.7. Works now for both versions. Thanks a lot.

**Comment by cwiechmann** [comment]
Thanks for the feedback @jschaefer77. I’m going to release a version 1.2.1 today/tomorrow which will include this fix.

**Comment by cwiechmann** [comment]
Fixed and part of release 1.2.1
